js/ui/validation_engine.js

```diff
@@ -451,7 +451,7 @@ const GroupConfig = Class.inherit({
             this._validationInfo.result.isValid = this._validationInfo.result.status === STATUS.valid;
             const res = extend({}, this._validationInfo.result, { complete: null });
             const deferred = this._validationInfo.deferred;
-            this._resetValidationInfo();
+            this._validationInfo.deferred = null;
```

Comment:
Why did you replace the this._resetValidationInfo(); method?

Reply:
`_validationInfo` contains info about group validation (several validators, many rules). We should not remove all this info on only single validator removing (this code is called from `removeRefisteredValidator`).

Particularly, it breaks the case when we dispose several integrated editors one by one